x-pack/plugin/core/src/main/resources/synthetics-template.json

```diff
@@ -0,0 +1,14 @@
+{
+  "index_patterns": ["synthetics-*-*"],
+  "priority": 100,
+  "data_stream": {},
+  "composed_of": [
+    "synthetics-mappings",
```

Comment:
@jpountz I remember you wanted to "not" use components here. Perhaps it is a good time to adjust?